mcp/sensors/star_camera_transmit.c

```diff
@@ -0,0 +1,396 @@
+/***************************************************************************
+ mcp: the TIM master control program
+ 
+ This software is copyright (C) 2002-2006 University of Toronto
+ 
+ This file is part of mcp.
+ 
+ mcp is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ at your option) any later version.
+ 
+ mcp is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ 
+ You should have received a copy of the GNU General Public License
+ along with mcp; if not, write to the Free Software Foundation, Inc.,
+ 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ 
+ created by Ian Lowe 5-23-23
+ **************************************************************************/
+
+
+/*************************************************************************
+ 
+ star_camera_transmit.c -- mcp code to create data streams to SC1 and SC2
+ 
+ *************************************************************************/
+
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netdb.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <unistd.h>
+#include <pthread.h>
+#include <signal.h>
+#include <errno.h>
+
+#include "star_camera_transmit.h"
+#include "command_struct.h"
+
+struct star_cam_capture sc1_command_packet;
+struct star_cam_capture sc2_command_packet;
+
```

Comment:
All functions in this file need doxygen-style commenting. See also my comment about including function declarations in the header: https://github.com/tim-balloon/TIMflight/pull/50#discussion_r1251139694